artifact.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -e
+
+wget -q -O artifact_none.sh 'https://raw.githubusercontent.com/tidepool-org/tools/ci-changes/artifact/artifact.sh'
```

Comment:
Why `artifact_none`? Isn't it producing a docker image (which is an artifact)?

Reply:
I will change the names in the next pass.  The name it cannot be is artifact.sh!

Reply:
Ah, of course.